cmd/system-probe/modules/traceroute.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build linux || windows
+//go:build linux || windows || darwin
```

Comment:
Probably a stupid question but: in `[sysprobe.go](https://github.com/DataDog/datadog-agent/pull/35966/files#diff-d4497ddb96f01b74afcbc2d08a7c1feb7d1e421732da596ea65c6d1a875121c7)`, we simply remove the line. Why here we add `darwin` and not just drop the line?

Reply:
I think it wasn't intentional (I modified one of the files and it was @AlexandreYang who changed the one you're pointing at I think) and it should be possible to drop it here without harm, too, I'll do that.